cmd/cli/user.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"github.com/moira-alert/moira"
+)
+
+func transferUserSubscriptionsAndContacts(database moira.Database, from, to string) error {
+	contactIDs, err := database.GetUserContactIDs(from)
+	if err != nil {
+		return err
+	}
+	contacts, err := database.GetContacts(contactIDs)
+	if err != nil {
+		return err
+	}
+	for _, contact := range contacts {
+		contact.User = to
+		if err = database.SaveContact(contact); err != nil {
+			return err
+		}
+	}
+
+	subscriptionIDs, err := database.GetUserSubscriptionIDs(*userDel)
```

Comment:
Ok we decided that passing arguments as `fromUser` and `toUser` will be more clear instead of using global variables. But what should we do with `userDel`? I think that this should be another one argument for function.

Reply:
corrected